extensions/data-transfer/portability-data-transfer-rememberthemilk/src/main/java/org/dataportabilityproject/transfer/rememberthemilk/model/tasks/AuthElement.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2018 The Data Transfer Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dataportabilityproject.transfer.rememberthemilk.model.tasks;
+
+import com.google.api.client.util.Key;
+
+/**
+ * Response from rtm.auth.getToken
+ *
+ * <p>Example: <?xml version='1.0' encoding='UTF-8'?><rsp
+ * stat="ok"><auth><token>0cefd358a1a6ec74b6a7a4f268681b27eb435916</token><perms>write</perms><user
+ * id="1650840" username="bwillard" fullname="Brian Willard"/></auth></rsp>
+ */
+public class AuthElement extends RememberTheMilkResponse {
```

Comment:
actually is this class needed? Don't see it used anywhere

Reply:
Woops youre right - this is probably for the AuthService, deleting it (and FrobResponse) for now and will re-add it with the implementation of the auth extension